packages/react/src/recipes/seo/Article/Article.tsx

```diff
@@ -405,223 +418,250 @@ export function Article({
-                                  <FAQ.Item>
-                                    <FAQ.Question>What is GitHub Enterprise Cloud?</FAQ.Question>
-                                    <FAQ.Answer>
-                                      <p>
-                                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
-                                        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.
-                                      </p>
-                                    </FAQ.Answer>
-                                  </FAQ.Item>
-                                </FAQ>
-
-                                <FAQ>
-                                  <FAQ.Heading>About GitHub Enterprise</FAQ.Heading>
-                                  <FAQ.Item>
-                                    <FAQ.Question>
-                                      What is the difference between GitHub and GitHub Enterprise?
-                                    </FAQ.Question>
-                                    <FAQ.Answer>
-                                      <p>
-                                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
-                                        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.
-                                      </p>
-                                    </FAQ.Answer>
-                                  </FAQ.Item>
-                                  <FAQ.Item>
-                                    <FAQ.Question>Why should organizations use GitHub Enterprise?</FAQ.Question>
-                                    <FAQ.Answer>
-                                      <p>
-                                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
-                                        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.
-                                      </p>
-                                    </FAQ.Answer>
-                                  </FAQ.Item>
-                                  <FAQ.Item>
-                                    <FAQ.Question>Who uses GitHub Enterprise?</FAQ.Question>
-                                    <FAQ.Answer>
-                                      <p>
-                                        Lorem ipsum dolor sit amet, consectetur adipiscing elit. In sapien sit
-                                        ullamcorper id. Aliquam luctus sed turpis felis nam pulvinar risus elementum.
-                                      </p>
-                                    </FAQ.Answer>
-                                  </FAQ.Item>
-                                </FAQ>
-                              </FAQGroup>
-                            </section>
-                          </Stack>
+                                    {line}
+                                  </span>{' '}
```

Comment:
```suggestion
                                  </span>
```

Reply:
So I struggled with this as, annoyingly, that space is actually necessary. I previously had a comment there explaining that, which might still be worth doing.

TL;DR is that I split the words in the title by spaces, then when reconstructing the lines I add the spaces back in between words. I also need to make sure I add a space in between the lines too, otherwise we end up with the last span of one line and the first span of the next squished together without a space between them, meaning that two words get combined into one.

Eg, without the space we have

```html
<span>This is the first line</span><span>and this is the second</span>
```

Which will render as either

```
This is the first lineand
this is the second
```

or

```
This is the first
lineand this is the second
```

Annoying and messy, I know. Open to alternate approaches, and at the very least I can add a comment explaining this.

Reply:
4f4e4971ac25106d2350acc7d6a9d6250bb34871

Reply:
2e3b4b1751fb52a5fc01103452c89d2cffdfd8e1